mucommander-viewer-text/src/main/java/com/mucommander/viewer/text/TextEditorImpl.java

```diff
@@ -98,7 +103,28 @@ public Insets getInsets() {
                 return new Insets(0, 3, 4, 3);
             }
         };
+        int linkScanningMask = OsFamily.MAC_OS.isCurrent() ? InputEvent.META_DOWN_MASK : InputEvent.CTRL_DOWN_MASK;
+        textArea.setLinkScanningMask(linkScanningMask);
+        textArea.addKeyListener(new KeyAdapter() {
+            @Override
+            public void keyPressed(KeyEvent e) {
+                if (textArea.getHyperlinksEnabled() && (e.getModifiersEx() & linkScanningMask) == linkScanningMask)
+                    fakeMouseMove(e);
+            }
+
+            @Override
+            public void keyReleased(KeyEvent e) {
+                fakeMouseMove(e);
+            }
 
+            private void fakeMouseMove(KeyEvent e) {
```

Comment:
If I understand it correctly, this is an attempt to workaround the problem on Ventura?
If so, wouldn't be helpful to add a comment with a link to #811 ?

Reply:
it happened to me on Monterey - one had to move the cursor while meta/ctrl was pressed in order to get the 'hand' cursor and to have the url rendered as a link, which make sense when you look at the code inside `RSyntaxTextArea` that is responsible for that. so that's not a workaround but rather an enhancement so one wouldn't have to move the mouse - the aforementioned behavior would happen also when the cursor is placed on a link and then meta/ctrl is pressed

Reply:
@ahadas - have you tried to run RSyntaxTextArea demo app and see if you reproduce this? If so, then maybe its worth to report an bug to RSyntaxTextArea project?

Reply:
nope, I didn't try with the demo app but I'd be really surprised if it works there - filing a bug or even contributing the change to the RSyntaxTextArea project makes a lot of sense